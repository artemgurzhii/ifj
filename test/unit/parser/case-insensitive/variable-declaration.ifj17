diM Foo AS  stRiNg

Dim fOo1 As DouBle

DiM foo2 aS IntegEr
