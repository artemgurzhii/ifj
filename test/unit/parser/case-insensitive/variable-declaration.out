(dim
  (decl
    (id foo)
    : (id string)))

(dim
  (decl
    (id foo1)
    : (id double)))

(dim
  (decl
    (id foo2)
    : (id integer)))
